src/integration_test.ts

```diff
@@ -40,7 +40,7 @@ describe('FullRequest', () => {
 
             const list = await k8sApi.listNamespacedPod({ namespace: 'default' });
 
-            return deepEqual(list, result);
+            deepEqual(list, result);
```

Comment:
Was the return here unnecessary? I'm guessing that deepEqual will throw and fail the test if these two object don't match?

Reply:
Yes, that `return` was superfluous. The return type is simply `void` and so it kinda makes no difference.

Reply:
Can you move this to a separate PR.